### DISCUSSION TITLE
ZR button shortcuts don't work

### DISCUSSION THREAD

**Discussion by Lollok007** [leoetlino/project-restoration#249] [Q&A]
The ZR button, and all its shortcuts, won't work. I'm playing on the android citra MMJ, the controller is configured correctly, and it works in other games. The mod works just fine apart from this problem

**Comment by andymerskin** [comment]
I'm experience the same thing on Citra (Windows) -- pressing Select doesn't open the Items menu, and the ZR shortcuts aren't working either. I saw that the mod is working since it made the noise when loading the game, so I'm not sure what's wrong.

**Comment by DrShadox** [comment]
I don't know if you tried since , but it work for me currently

if its still dont work , maybe its a controller issue ?

**Comment by Euniemeansme** [comment]
If anyone is still having problems with this, you must have New 3ds mode enabled in Citra settings, and the Circle Pad Pro options enabled in game, otherwise they won't work.
